### PR TITLE
Me look prettier one day

### PR DESCRIPTION
**Pull request by srkirkland**

### CODE REVIEW

**Review by jSylvestre — APPROVED** [review]
YOLO

**Review by sprucely — APPROVED** [review]
☝️

File: Harvest.Web/ClientApp/src/FormValidation/UseInputValidator.tsx [hunk 1]
```diff
@@ -29,7 +48,9 @@ export function useInputValidator<T>(
 
   useEffect(() => {
     const errorCount = Object.values(errors).filter(notEmptyOrFalsey).length;
-    const previousErrorCount = Object.values(previousErrors || {} as Record<TKey, string>).filter(notEmptyOrFalsey).length;
+    const previousErrorCount = Object.values(
+      previousErrors || ({} as Record<TKey, string>)
```
Comment (jSylvestre) [inline Harvest.Web/ClientApp/src/FormValidation/UseInputValidator.tsx:52]: This added an extra ( but I think that is correct?

File: Harvest.Web/ClientApp/src/schemas.ts [hunk 2]
```diff
@@ -46,22 +51,37 @@ export const ticketSchema: SchemaOf<TicketInput> = yup.object().shape({
...
-  ),
+  quantity: yup
+    .number()
+    .required()
+    .typeError(ErrorMessages.WorkItemUnit)
+    .positive(ErrorMessages.WorkItemUnit)
+    .test(
+      "maxDigitsAfterDecimal",
+      ErrorMessages.WorkItemQuantityDecimalPlaces,
+      (number) => Number.isInteger((number || 0) * 10 ** 2)
```
Comment (jSylvestre) [inline Harvest.Web/ClientApp/src/schemas.ts:73]: I'm guessing this is right. Kind of bugs me that it removes the unnecessary ()

File: Harvest.Web/ClientApp/src/FormValidation/UseInputValidator.tsx (hunk 1 above)
Comment (sprucely) [reply Harvest.Web/ClientApp/src/FormValidation/UseInputValidator.tsx:52]: Yeah, prettier likes to put casts inside parenthesis.

**Pull request merged**